pdbufr/bufr_read.py

```diff
@@ -11,6 +11,7 @@
 
 import eccodes  # type: ignore
 import pandas as pd  # type: ignore
+import geopandas as gpd  # type: ignore
```

Comment:
Please make the import optional, for example using the following (untested)

```suggestion
try:
    import geopandas as gpd  # type: ignore
    HAS_GEOPANDAS = True
except ModuleNotFoundError:
    HAS_GEOPANDAS = False
```

Reply:
Yes, reproducible - included (see below)